integration-cli/docker_cli_events_test.go

```diff
@@ -628,7 +628,7 @@ func (s *DockerSuite) TestEventsCopy(c *check.C) {
 		c.Fatalf("Failed getting file:%q\n%q", out, err)
 	}
 
-	cmd = exec.Command(dockerBinary, "events", "-f", "container=cptest", "--until="+strconv.Itoa(int(since)))
+	cmd = exec.Command(dockerBinary, "events", "--since=0", "-f", "container=cptest", "--until="+strconv.Itoa(int(since)))
```

Comment:
But it doesn't say what happens when there is no timestamp given.  Show all or show from "now".  Just something simple like:
```
When no '--since' value is specified, the default value is the current time.
```